### ISSUE TITLE
ModuleManager manual suppression causes a bug

### ISSUE THREAD

**Issue by Maverickwoe** [KSP-CKAN/CKAN#847]
Hi

I'm new using CKAN.
It's great but I think I found an exception not covered.
I'm trying to manually remove, one by one all the modules installed without CKAN before.
It works fine for everything excepted the ModuleManagerxxx.dll file.
If I delete it and refresh CKAN, I instantly get this error:

The following inconsistencies were found:
RasterPropMonitor requires ModuleManager but nothing provides it
 * CommunityTechTree requires ModuleManager but nothing provides it
 * RasterPropMonitor-Core requires ModuleManager but nothing provides it   à CKAN.SanityChecker.EnforceConsistency(IEnumerable`1 modules, IEnumerable`1 dlls)
   à CKAN.RegistryManager.Save(Boolean enforceConsistency)
   à CKAN.KSP.ScanGameData()
   à CKAN.Main.UpdateRepo()
   à System.Windows.Forms.ToolStripMenuItem.OnClick(EventArgs e)
   à System.Windows.Forms.ToolStripItem.HandleClick(EventArgs e)
   à System.Windows.Forms.ToolStripItem.HandleMouseUp(MouseEventArgs e)
   à System.Windows.Forms.ToolStrip.OnMouseUp(MouseEventArgs mea)
   à System.Windows.Forms.Control.WmMouseUp(Message& m, MouseButtons button, Int32 clicks)
   à System.Windows.Forms.Control.WndProc(Message& m)
   à System.Windows.Forms.ToolStrip.WndProc(Message& m)
   à System.Windows.Forms.MenuStrip.WndProc(Message& m)
   à System.Windows.Forms.NativeWindow.Callback(IntPtr hWnd, Int32 msg, IntPtr wparam, IntPtr lparam)

So, I cannot ask CKAN to automaticaly update this file as I can t delete it without making CKAN crash.

You might want to have a look

cheers
and thanks for the lovely program

**Comment by Maverickwoe** [comment]
Does anyone knows a workaround for this ?

**Comment by Dazpoet** [comment]
http://forum.kerbalspaceprogram.com/threads/100067-The-Comprehensive-Kerbal-Archive-Network-%28CKAN%29-Package-Manager-v1-6-16-13-May-2015?p=1933137&viewfull=1#post1933137

**Comment by Dazpoet** [comment]
I'll take the no answer as a "the above solved the problem" and close this. If not please comment and we can reopen it.

**Comment by Maverickwoe** [comment]
Sorry, I thought no one answered the thread, didn t see the notification earlier.
I just followed the link and rummaged through the post.
It worked

Module manager was actually required for most of the mods I had and a "scan" command in the prompt revealed which one it was dependant to when the dll was missing.
I just had to remove these mods and let cKan reinstall module manager when reinstalling the same mods again.

Quite happy it's handled by CKan now !
Cheers
